airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1746,6 +1780,9 @@
   icon: snowflake.svg
   sourceType: database
   releaseStage: alpha
+  allowedHosts:
+    hosts:
+      - "${host}"
```

Comment:
I think it might be called "account_name" for snowflake:
<img width="626" alt="Screenshot 2023-02-24 at 3 20 40 PM" src="https://user-images.githubusercontent.com/303226/221319663-2fa5f1d5-060f-4d6b-9a93-10dab44307b5.png">

Reply:
Nope! I was wrong - it's still "host":

https://github.com/airbytehq/airbyte/blob/7da6a3bb77c4af9db10afda358f81cac48728869/airbyte-config/init/src/main/resources/seed/source_specs.yaml#L14068-L14075

Reply:
Thanks for pointing me to `source_specs.yaml`. Turns out it is not `host` name in all places.